dashboard/src/actions/apps.ts

```diff
@@ -74,7 +134,7 @@ export function deleteApp(
   };
 }
 
-export function fetchApps(
+export function fetchAppsWithUpdatesInfo(
```

Comment:
Although we don't use it separately, there is now no way to just `fetchApps` by themselves. I would keep the `fetchApps` action and change it to return `dispatch(receiveAppList(apps));`, and then dispatch `fetchApps` from `fetchAppsWithUpdatesInfo` and await on it. See the composition examples in https://github.com/reduxjs/redux-thunk#composition, the actions are designed to return the data that is also dispatched, so that other actions can be built on top of them.